procedures/steering-meeting.md

```diff
@@ -0,0 +1,97 @@
+# Steering meeting
+
+The "steering meeting" is a weekly meeting dedicated to planning and
+high-level discussion. The meeting operates on a repeating schedule:
+
+- Week 1: Planning
+- Week 2: Technical or non-technical discussion
+- Week 3: Technical or non-technical discussion
+- Week 4: Non-technical discussion
+
+The first meeting of the 4-week cycle is used for **planning**. The
+primary purpose of this meeting is to **select the topics for the next
+three meetings**. The topics are selected from a set of topic
+proposals, which must be uploaded and available for perusal before the
+meeting starts. The planning meeting is also an opportunity to check
+on the "overall balance" of our priorities.
+
+The remaining meetings are used for design or general discussion.
+Weeks 2 and 3 can be used for **technical** or **non-technical**
+discussion; it is also possible to use both weeks to discuss the same
+topic, if that topic is complex. **Week 4 is reserved for
+non-technical topics**, so as to ensure that we are keeping an eye on
+the overall health and functioning of the team.
+
+## Announcing the schedule
+
+After each planning meeting, the topics for the next three weeks will
+be posted on the README.md of the main repository and also sent a
+dedicated internals thread.
+
+## Examples of good candidates for discussing at the steering meeting
+
+Here are some examples of possible technical topics that would be
+suitable for the steering meeting:
+
+- A working group has an idea to refactor the HIR to make some part of their
+  job easier. They have sketched out a proposal and would like feedback.
+- Someone has encountered a problem that is really hard to solve with
+  the existing data structures. They would like feedback on a good
+  solution to their problem.
+- Someone has done major refactoring work on a PR and they would like
+  to be able to explain the work they did and request review.
+
+Steering meetings are also a good place to discuss other kinds of proposals:
+  
+- A proposal to move some part of the compiler into an out-of-tree crate.
+- A proposal to start a new working group.
+
+Note that a steering meeting is **not** required to create a new
+working group or an out-of-tree crate, but it can be useful if the
+proposal is complex or controversial, and you would like a dedicated
+time to talk out the plans in more detail.
+
+## Criteria for selection
+
+When deciding the topics for upcoming meetings, we must balance a number of things:
+
+- We don't want to spend time on design work unless there are known
+  people who will implement it and support it.
```

Comment:
Maybe explicitly mention that this requires more than one person. There needs to be at least a second person, with enough time and background knowledge to do the reviewing.